nvflare/fuel/flare_api/flare_api.py

```diff
@@ -153,7 +153,10 @@ def _do_command(self, command: str, enforce_meta=True, props=None):
             elif cmd_status == MetaStatusValue.INTERNAL_ERROR:
                 raise InternalError(f"server internal error: {info}")
             elif cmd_status == MetaStatusValue.INVALID_TARGET:
-                raise InvalidTarget(info)
+                if info == "no clients available":
```

Comment:
Use string comparison might be prone to errors when the source change text, where does this come from maybe we can add a comment here

Reply:
Changed to use defined keywords